frontend/src/app/application-forms/applications.module.ts

```diff
@@ -53,6 +56,7 @@ import 'intersection-observer';
     TemporaryOutfittersComponent,
     TempOutfitterLeftNavComponent,
     TemporaryOutfittersLearnMoreComponent,
+    TreeApplicationFormComponent,
```

Comment:
@sethalt I know we had talked about this before, but do we maybe want to think about where the learn more's load, as it seems like maybe a kick to performance to have to load the temp outfitter/ noncommerical learnmore for xmas tree. Or am I mistaken that that won't have to be loaded when loading the xmas tree form?